test/services/candlepin/message_handler_test.rb

```diff
@@ -145,4 +124,43 @@ def test_delete_pool_on_null
       assert_empty Katello::Pool.find_by(:cp_id => @pool_id)
     end
   end
+
+  class OwnerContentAccessModeModifiedTest < MessageHandlerTestBase
+    let(:event_name) { 'owner_content_access_mode.modified' }
+
+    def setup
+      @org = get_organization(:empty_organization)
+      Katello::Resources::Candlepin::Owner.expects(:all).returns(
+        [
+          {
+            'displayName' => @org.name,
+            'key' => @org.label
+          }
+        ]
+      )
+      super
+    end
+
+    def test_sca_enabled
+      Katello::HostStatusManager.expects(:clear_syspurpose_status)
+      Katello::HostStatusManager.expects(:update_subscription_status_to_sca)
+      Organization.any_instance.expects(:simple_content_access?).with(cached: false)
+
+      @handler.handle_content_access_mode_modified
+    end
+
+    def test_sca_disabled
+      Katello::HostStatusManager.expects(:clear_syspurpose_status).never
+      Katello::HostStatusManager.expects(:update_subscription_status_to_sca).never
+      Organization.any_instance.expects(:simple_content_access?).with(cached: false)
+      @handler.expects(:event_data).returns('contentAccessMode' => 'entitlement').twice
+
+      @org.hosts.joins(:subscription_facet).count.times do
```

Comment:
Can this number change for each test run?  If not, should probably be hard-coded and avoid the DB query

Reply:
It could change since this organization is one of our fixtures